src/plugins/columnSorting/columnSorting.js

```diff
@@ -78,6 +78,8 @@ class ColumnSorting extends BasePlugin {
      * @type {Boolean}
      */
     this.sortEmptyCells = false;
+    // It blocks the plugin translation, this flag is checked inside `onModifyRow` listener.
+    this.blockPluginTranslation = true;
```

Comment:
Could you add proper JSDoc tags? (I suppose that here you should add @private tag)